Monika After Story/game/zz_selector.rpy

```diff
@@ -2754,6 +2754,22 @@ label mas_selector_sidebar_select_clothes(items, preview_selections=True, only_u
 # [MONSEL]
 
 #### Begin monika clothes topics
+init 200 python:
```

Comment:
since `event_clothes` and `gifted_clothes` are related to selectables, make this init in the `selspr` store
```suggestion
init 200 python in mas_selspr:
```